reco_utils/recommender/ncf/dataset.py

```diff
@@ -26,12 +24,25 @@ def __init__(
         col_item=DEFAULT_ITEM_COL,
         col_rating=DEFAULT_RATING_COL,
         col_timestamp=DEFAULT_TIMESTAMP_COL,
+        implicit=True,
         seed=42,
     ):
+        """Constructor 
+        
+        Args:
+            train (pd.DataFrame): Training data with at least columns (col_user, col_item, col_rating).
+            test (pd.DataFrame): Test data with at least columns (col_user, col_item, col_rating). test can be None, 
+                if so, we only process the training data.
+            n_neg (int): Number of negative samples for training set.
+            n_neg_test (int): Number of negative samples for test set.
+            col_user (str): User column name.
+            col_item (str): Item column name.
+            col_rating (str): Rating column name. 
+            col_timestamp (str): Timestamp column name.
+            implicit (bool): If true, set rating > 0 to rating = 1. 
```

Comment:
Do you mean "If true, set rating > 0 else set rating = 1." ?

Reply:
changed